### PR TITLE
Add instructions link to enable virtualisation

### PR DESCRIPTION
**Pull request by Josehower**
### Proposed changes

I just found it hard to find good instructions to enable virtualization from the UEFI or BIOS. So I just added a useful link. I used the archived version so is not going to get down if the resource change.

### CODE REVIEW

File: desktop/windows/troubleshoot.md [hunk 1]
```diff
@@ -274,7 +274,7 @@ Your machine must have the following features for Docker Desktop to function cor
 
 1. Virtual Machine Platform
 2. [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10){:target="_blank" rel="noopener" class="_"}
-3. Virtualization enabled in the BIOS
+3. [Virtualization enabled in the BIOS](https://archive.ph/0m7oE){:target="_blank" rel="noopener" class="_"}
```
Comment (thaJeztah) [inline desktop/windows/troubleshoot.md:277]: Looks like this URL is a mirror of the original content (with lots of advertising / tracking).

The original content comes from https://github.com/ucberkeley/bce-documentation/blob/gh-pages/enabling-virtualization-in-your-pc-bios.md, and from that repository, it looks like the official URL for that page is https://bce.berkeley.edu/enabling-virtualization-in-your-pc-bios.html

```suggestion
3. [Virtualization enabled in the BIOS](https://bce.berkeley.edu/enabling-virtualization-in-your-pc-bios.html){:target="_blank" rel="noopener" class="_"}
```

File: desktop/windows/troubleshoot.md [hunk 2]
```diff
@@ -285,7 +285,7 @@ On Windows 10 Pro or Enterprise, you can also use Hyper-V with the following fea
 
 1. [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/hyper-v-technology-overview){:target="_blank" rel="noopener" class="_"}
    installed and working
-2. Virtualization enabled in the BIOS
+2. [Virtualization enabled in the BIOS](https://archive.ph/0m7oE){:target="_blank" rel="noopener" class="_"}
```
Comment (thaJeztah) [inline desktop/windows/troubleshoot.md:288]: Same here;

```suggestion
2. [Virtualization enabled in the BIOS](https://bce.berkeley.edu/enabling-virtualization-in-your-pc-bios.html){:target="_blank" rel="noopener" class="_"}
```

**Comment by Josehower** [comment]
HI @thaJeztah,

Just checking if you had some time to see this, really appreciated it.

thanks

**Comment by thaJeztah** [comment]
Thanks for the ping! I did a quick rebase and squash to get a fresh run of CI

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Thanks!

**Pull request merged**